src/ontology/Makefile

```diff
@@ -216,6 +216,11 @@ VQUERIES = $(foreach V,$(VCHECKS),$(SPARQLDIR)/$V-violation.sparql)
 sparql_test: $(SRC)
 	$(ROBOT) verify -i $< --queries $(VQUERIES) -O reports/
 
+ensure_terms_%: %
+	$(ROBOT) query -i $< --query ../sparql/ensure-terms-$<.sparql ensure-terms-$<.txt && grep -q true ensure-terms-$<.txt
+
+all: ensure_terms_hpo_slim.owl
```

Comment:
this does not look right.
First of all, there is an `all` rule on line 33.
Second, the dependency seems wrong.